src/Common/tests/GenStaticTestRunner/GenStaticTestRunner.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <LangVersion>Preview</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NoWarn />
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Program.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.Compilers" Version="2.10.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.10.0" />
+    <PackageReference Include="xunit.abstractions" Version="2.0.3" />
+    <PackageReference Include="xunit.console" Version="2.4.1" />
```

Comment:
Do we want to use the Xunit versions defined in Versions.props? Or do we want to depend on static versions of it?

Reply:
I was thinking we want to keep the tool separate, and if things change in the dependency, it'll likely require more fixup in the tool because it replaces the entire execution mechanism. But I could be convinced otherwise.

Reply:
Either way it is fine, I was mentioning it to be consistent and avoid having to update it manually later and investigating issues (if a new feature comes along and we start using it in corefx, then the tool could fail because of it depending on an older version of xunit) -- which has happened before. Also, depending on a version that can easily change could break the tool, but it is always good to be in the latest version IMHO.

Also, I don't think we're going to update the version anytime soon, since we've forked the runner again until we move to the new executor, that said, avoiding duplication would be nice.

Reply:
> Either way it is fine

Ok.  Let's stick with this for now.  We can always integrate it into the build system later if this proves problematic.  My expectation though is that any updates to xunit that result in new features being used in corefx tests will also require code changes to this tool, so just having the version bumped automatically wouldn't end up being useful.  I could be wrong, of course.